package.json

```diff
@@ -13,9 +13,9 @@
   "readmeFilename": "README.md",
   "gitHead": "eb6f38decdf15fc18625bf60a602db2937feb229",
   "scripts": {
-    "build": "$(npm bin)/grunt",
-    "release": "$(npm bin)/grunt release",
-    "watch": "$(npm bin)/grunt watch"
```

Comment:
Yes, I am on Windows.  Sorry, I didn't realize I had checked that change in (though I _think_ that should be a safe change to make the project work better cross-platform).